Update labels: blank-issue-template(role,size,feature,complexity as missing)

kiwookim opened the pull request:
Fixes #4503

### What changes did you make and why did you make them ?

  - added labels for blank-issue-template.md, so that we can avoid GitHub bot taking off the labels when devs add them to issues
  -  added URL of the issue template into the Resources section of the issue, replacing the text: https://github.com/kiwookim/website/issues/new?assignees=&labels=role+missing%2CFeature+Missing%2CComplexity%3A+Missing%2Csize%3A+missing&projects=&template=blank-issue-template.md&title=

### Screenshots of Proposed Changes Of The Website  (if any, please do not screen shot code changes)
<!-- Note, if your images are too big, use the <img src="" width="" length="" />  syntax instead of ![image](link) to format the images -->
<!-- If images are not loading properly, you might need to double check the syntax or add a newline after the closing </summary> tag -->

<details>
<summary>No visual changes to the website</summary>


</details>

kiwookim committed:
revert

kiwookim committed:
update labels again to get URL for PR

kiwookim committed:
added link of new issue template labels for blank-issue-template

matthewmpan commented:
Review ETA: 2 PM 5/20/23
Availability: 12-5 PM Saturday

matthewmpan reviewed (changes requested):
Hi @kiwookim Your branches are set up correctly and your feature branch is appropriately named. I would recommend giving your pull request a more descriptive title that highlights your changes. 

You correctly linked the issue and checked off all of the action items listed. However, there are two places in issue #4503 to replace existing [placeholders] with a link to the issue template in your repo. The first section is in the issue section **Link for Reviewers** and the second section is in your comment in checkbox 9. The link to your issue template should look something like this: [https://github.com/kiwookim/website/issues/new?assignees=&labels=role+missing%2CFeature+Missing%2CComplexity%3A+Missing%2Csize%3A+missing&projects=&template=blank-issue-template.md&title=](url) 

Lastly, revert your last commit aa81ad32a6af3d14f834e2df579dacd2b81a320b. Line 18 of `.github/ISSUE_TEMPLATE/blank-issue-template.md` is meant to serve as part of the issue template for new issues, letting issue creators know where to add additional Resources/Instructions.

93Belen commented:
Availability: Tuesday, Thursday and Sunday from 7am to 5pm
ETA: 05/23/23 by 6pm

93Belen reviewed (changes requested):
Hey, great job on changing the labels! You did the pull request from the correct branch and linked the issue correctly. Nice work!

I noticed that you added the correct labels, but you also included some code on lines 19 and 20 that wasn't mentioned in the issue. 

Remember to include the link to the test repository in the placeholders so we can review the visual changes.

Also, I would mention what labels you added in the description and update the title for a more descriptive one.

Keep up the good work! Let me know if you have any questions or need help.

kiwookim committed:
removed unnecassary code and comment

kiwookim commented:
Hi, Matthew and Belen! Thank you for your reviews. I changed the title of the PR, added the URL, and removed unnecessary comments and code in the readme file. Please let me know if there is anything else. Thank you.

matthewmpan commented:
Review ETA: EOD 5/24/23
Availability: 12-5 PM M-F

93Belen commented:
Availability: Thursday and Sunday from 6am to 6pm
ETA: 5/25/23 by 10am

roslynwythe reviewed (changes requested):
Hi @kiwookim Thank you for your work on this issue, and in particular for following the test procedure and checking off those items. Please update the section of the PR that says "[REPLACE THIS TEXT WITH URL OF THE ISSUE TEMPLATE ON THE TEST REPO]" with the actual URL, for the convenience of the other reviewers.   I did follow the URL to the template and it looks fine to me. Everything else looks good as well: the branches are properly formed and the code change is clean.  So I'll approve this as soon as the URL is posted in the PR.

kiwookim commented:
> Hi @kiwookim Thank you for your work on this issue, and in particular for following the test procedure and checking off those items. Please update the section of the PR that says "[REPLACE THIS TEXT WITH URL OF THE ISSUE TEMPLATE ON THE TEST REPO]" with the actual URL, for the convenience of the other reviewers. I did follow the URL to the template and it looks fine to me. Everything else looks good as well: the branches are properly formed and the code change is clean. So I'll approve this as soon as the URL is posted in the PR.

Thank you Roslyn!

matthewmpan reviewed (approved):
Hi @kiwookim Thanks for making the requested changes to the PR name, lines 18-20, and adding the reviewer link. I was able to confirm the changes in the issue template and everything looks good. Great work!

roslynwythe reviewed (approved):
Hi @kiwookim thank you for making the request changes.

93Belen commented:
Hi @kiwookim! Since I was going to ask for the same changes as @roslynwythe, I am just going to wait for those changes before reviewing again :)

93Belen reviewed (approved):
Hi! Thanks for adding the link! nice job addressing the requested changes!
